make/default.example.mk.help.log

```diff
@@ -1,13 +1,26 @@
 The following make targets are available:
 all
 build
+:cd=\E[J:ce=\E[K:cl=\E[H\E[J:cm=\E[%i%d;%dH:ct=\E[3g:
```

Comment:
try running it in centos container

Reply:
Done.  If this is a requirement for running `make update`, maybe check `/etc/redhat-release` and error out?

Reply:
Even after running it in a centos container, it's still failing the verify job. Do you want to give a try and propose the commit to my PR branch?

Also, can this be simplified so that it just works?

Reply:
Can you update it with what you get when running this inside a container? like
```
podman run -it --rm --workdir=/go/src/github.com/openshift/build-machinery-go -v `pwd`:/go/src/github.com/openshift/build-machinery-go registry.svc.ci.openshift.org/ocp/builder:golang-1.13 make update
```
 `:cd=\E[J:ce=\E[K:cl=\E[H\E[J:cm=\E[%i%d;%dH:ct=\E[3g:` special chars seem to come from your shell.

> Also, can this be simplified so that it just works?
Not sure, running inside container is the best  I come up with. It's not like make has unit test framework built in. I can't even fail on warnings.

I wanted to add option running this in a container by default but it got stuck in https://github.com/openshift/library-go/pull/543

Reply:
Yes, that worked. And the result was a one line change.